src/utils.js

```diff
@@ -0,0 +1,93 @@
+// user selects one of these options when first adding an item to a list
+export const radioButtonOptions = [
+  {
+    option: 'soon',
+    value: 7,
+    defaultChecked: true,
+  },
+  {
+    option: 'kind of soon',
+    value: 14,
+    defaultChecked: false,
+  },
+  {
+    option: 'not soon',
+    value: 30,
+    defaultChecked: false,
+  },
+];
+
+//used to normalize item names when added by a user
+export const removePunctuation = (string) => {
+  const punctuationlessString = string
+    .toLowerCase()
+    .replace(/[^\w\s]|_/g, '')
+    .split(' ')
+    .join('');
+
+  return punctuationlessString;
+};
+
+// general variables used for time calculations
+export const currentTime = Date.now();
+export const oneDay = 86400000;
+
+// used to determine if a checkbox can be unchecked
+export const within24hours = (date) => {
+  let timeCheck = false;
+
+  const gap = currentTime - date;
+  if (gap < oneDay) {
+    timeCheck = true;
+  }
+  return timeCheck;
+};
+
+// used in ListLayout to determine how items are sorted by status
+export const groups = [
```

Comment:
I'd move this one to configuration.js (or similar name), and rename it so it's obvious what it does - something like todoItemGroups